spec/src/main/asciidoc/introduction.adoc

```diff
@@ -0,0 +1,32 @@
+= Introduction
+
+This section introduces the Jakarta Config Specification. Unless noted otherwise, this section is non-normative.
+
+CAUTION: You are reading an incomplete DRAFT specification.  The following is subject to change.
+
+== Overview
```

Comment:
It seems this section is very similar to the `Goals` section. Maybe they can be merged?